.changeset/unlucky-jobs-report.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-permission-backend': patch
+---
+
+Migrated to use the new auth services introduced in [BEP-0003](https://github.com/backstage/backstage/blob/master/beps/0003-auth-architecture-evolution/README.md).
+
+The `createRouter` function now has an optional `identity` argument, and instead gained the new `auth`, `httpAuth`, and `userInfo` arguments that should be set to the values of those respective `coreServices`. For users of the new backend system, this happens automatically without code changes.
```

Comment:
```suggestion
Migrated to use the new auth services introduced in [BEP-0003](https://github.com/backstage/backstage/blob/master/beps/0003-auth-architecture-evolution/README.md).

The `createRouter` function no accepts `auth`, `httpAuth` and `userInfo` options. Theses are used internally to support the new backend system, and can be ignored.
```

If you want to call out the new options I'm figuring it's best to explicitly say that they can be ignored? 😅 nobody is supposed to be trying to pass those in so I'm worried it's just gonna cause confusion tbh